google/cloud/speech/v1/cloud_speech_gapic.yaml

```diff
@@ -0,0 +1,52 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
+language_settings:
+  java:
+    package_name: com.google.cloud.speech.v1
```

Comment:
Done. I'll create a fix for language API -- but I'd want to say this is a bug of configgen (cc: @shinfan)